code/modules/admin/verbs/podlauncher.dm

```diff
@@ -713,25 +713,27 @@
 		qdel(M)
 
 
-/datum/podlauncher/proc/supplypod_punish_log(list/whoDyin)
+/datum/podlauncher/proc/supplypod_log(list/targets, atom/target)
 	var/podString = effectBurst ? "5 pods" : "a pod"
 	var/whomString = ""
-	if(LAZYLEN(whoDyin))
-		for(var/mob/living/M in whoDyin)
+	if(LAZYLEN(targets))
```

Comment:
```suggestion
	if(length(targets))
```
I have no idea who added this monstrosity to  our codebase, but it must be purged.

Reply:
It must not. It's actually better to use it since if another method ever becomes more useful there's only one place to change it. All of these macros are genius.